dcos/dcos-ui
# DCOS-55845 fullscreen group modal

Introduced a feature flagged full-screen modal to be used when creating
a top-level group. Current implementation only has group create with
name using data-layer. This is the first step in supporting creating a
group with quota.

## Testing


In `Config.dev` set `useUIConfigFixtures: true` and 
```
  uiConfigurationFixture: {
    uiConfiguration: {
      features: {     
        quota: true
...
```

1. Go to `/services/overview`
2. Click `Create Group`
3. *Verify full screen modal is displayed*
4. Set a name for you group
5. Click Save
6. Verify Group is created.

## Trade-offs



## Dependencies



## Screenshots


<img width="1081" alt="image" src="https://user-images.githubusercontent.com/1972968/61668388-2364d180-aca2-11e9-979c-67708a2b1310.png">
<img width="1081" alt="image" src="https://user-images.githubusercontent.com/1972968/61668403-2bbd0c80-aca2-11e9-8c52-3564d9d921af.png">
<img width="1081" alt="image" src="https://user-images.githubusercontent.com/1972968/61668416-35df0b00-aca2-11e9-8796-37b21aaddef9.png">
